integration-test/src/main/java/org/commonjava/maven/ext/integrationtest/invoker/DefaultExecutionParser.java

```diff
@@ -40,12 +40,7 @@
         add( ExecutionParser.SYSTEM_PROPERTIES_HANDLER );
     }};
 
-    private List<ExecutionParserHandler> handlers;
-
-    public DefaultExecutionParser()
-    {
-        this( new ArrayList<ExecutionParserHandler>() );
-    }
+    private final List<ExecutionParserHandler> handlers;
```

Comment:
Would it make sense to use `Collections.unmodifiableList()` here, too?

Reply:
I've taken another look and simplified the code more. It now implicitly uses a fixed size list.